backend/sirius-web-emf/src/main/java/org/eclipse/sirius/web/emf/compatibility/modeloperations/ExternalJavaActionOperationHandler.java

```diff
@@ -0,0 +1,72 @@
+package org.eclipse.sirius.web.emf.compatibility.modeloperations;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.sirius.tools.api.ui.IExternalJavaAction;
+import org.eclipse.sirius.viewpoint.description.tool.ExternalJavaAction;
+import org.eclipse.sirius.viewpoint.description.tool.ExternalJavaActionParameter;
+import org.eclipse.sirius.viewpoint.description.tool.ModelOperation;
+import org.eclipse.sirius.web.compat.api.IModelOperationHandler;
+import org.eclipse.sirius.web.compat.utils.BeanUtil;
+import org.eclipse.sirius.web.emf.compatibility.ExternalJavaActionManager;
+import org.eclipse.sirius.web.interpreter.AQLInterpreter;
+import org.eclipse.sirius.web.representations.Status;
+import org.eclipse.sirius.web.representations.VariableManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handle the {@link ExternalJavaAction} model operation.
+ *
+ * @author Charles Wu
+ */
+public class ExternalJavaActionOperationHandler implements IModelOperationHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(MoveElementOperationHandler.class);
+
+    private final AQLInterpreter interpreter;
+    private final ChildModelOperationHandler childModelOperationHandler;
+    private final ExternalJavaAction externalJavaAction;
+
+    public ExternalJavaActionOperationHandler(AQLInterpreter interpreter,
+                                              ChildModelOperationHandler childModelOperationHandler,
+                                              ExternalJavaAction externalJavaAction) {
+        this.interpreter = Objects.requireNonNull(interpreter);
+        this.childModelOperationHandler = Objects.requireNonNull(childModelOperationHandler);
+        this.externalJavaAction = Objects.requireNonNull(externalJavaAction);
+    }
```

Comment:
While this is not a big deal, this code is not formatted using our code formatter.